api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -1140,15 +1140,10 @@ components:
       type: string
       enum:
         - OpenShift Container Platform
-        - RHEL Compute Node
```

Comment:
Card has  RHEL Workstation and Compute Node as Count but don’t show, does that mean we want to keep them so that user can request data using api calls atelast ?